bin/app.js

```diff
@@ -1,22 +1,4 @@
 #!/usr/bin/env node
+const sub = require('./sub.js');
 
-const shell = require('shelljs');
-const inquirer = require('inquirer');
-const chalk = require('chalk');
-const header = require('../lib/header');
-const question = require('../lib/question');
-const create = require('../lib/create');
-
-const main = async () => {
-  header.init();
-
-  const input = await question.questions();
-  // console.log(input);
-  const { filename, extension } = input;
-  // console.log(filename);
-  create.create(filename, extension);
-
-  console.log(chalk.blue('File created successfully') + chalk.red('!!!'));
-};
-
-main();
+sub.main();
```

Comment:
I dont think the new file `sub.js` is necessary. you can declare the same in the `app.js`

Reply:
Yeah, I don't remember why I did that. I put it back in app.js with the latest commit.